README.md

```diff
@@ -1,28 +1,41 @@
-# Metal³ Documentation
+<p align="center"><img alt="Metal³" src="./images/metal3.png" /></p>
 
-![Metal³ Logo](images/metal3.png)
+# Metal³
+
+## Useful links
+
+* [Quick start](URL)
```

Comment:
Does the `URL` part here need to be replaced?

Reply:
Maybe this section should come after the "What is" section?

Reply:
Sorry, for the missing URL. Fixed that.